AEPAudience.podspec

```diff
@@ -18,7 +18,6 @@ Pod::Spec.new do |s|
 
   s.dependency 'AEPCore', '>= 5.0.0', '< 6.0.0'
```

Comment:
Should the AEPCore dependency minimum get changed to 5.1.0?

Reply:
I will run the script for that :)